### DISCUSSION TITLE
401 Client error

### DISCUSSION THREAD

**Discussion by SameerShanbhogue** [Project-MONAI/model-zoo#287] [Q&A] [answered]
monailabel server not able to start while using app monaibundle.

requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://api.github.com/repos/Project-MONAI/model-zoo/releases

**Comment by tangy5** [comment, marked as answer]
Hi @SameerShanbhogue , can you try add this in your monailabel start_server command. 

`--conf auth_token <Your Github Auth Token>`

I guess there is a authorization or rate limit issue for Github.

**Reply by SameerShanbhogue** [reply]
Thank you tangy, it works
